components/graphs/Graph.tsx

```diff
@@ -114,7 +135,51 @@ const Graph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
     getXTickValues(clearSkyEstimateTrimmed as ClearSkyDataPoint[], 5);
 
   return (
-    <div className="w-full h-[260px] bg-ocf-black-500 rounded-2xl p-1">
+    <div className="w-full h-[290px] bg-ocf-black-500 rounded-2xl p-1">
+      <div className="mt-3 ml-1">
+        <label className="mx-2">
+          <input
+            className="hidden peer"
+            type="radio"
+            name="1H"
+            id="1H"
+            value="1"
+            checked={timeRange === '1'}
```

Comment:
Make sure to replace these comparisons with the number `1` (and others below), after changing the state variable to store a number.